blocks/identity-block/features/account-management/_children/EmailEditableFieldContainer.jsx

```diff
@@ -0,0 +1,54 @@
+import React, { useEffect, useState } from 'react';
+import { useFusionContext } from 'fusion:context';
+import getProperties from 'fusion:properties';
+import getTranslatedPhrases from 'fusion:intl';
+import { useIdentity } from '../../..';
+import EditableFieldPresentational from '../../../components/EditableFormInputField';
+
+function EmailEditableFieldContainer() {
+  const [email, setEmail] = useState('');
+  const [error, setError] = useState('');
+
+  const { arcSite, isAdmin } = useFusionContext();
+  const { locale } = getProperties(arcSite);
+  const phrases = getTranslatedPhrases(locale);
+
+  const formEmailLabel = phrases.t('identity-block.email');
+  const emailRequirements = phrases.t('identity-block.email-requirements');
+
+  const { Identity } = useIdentity();
+
+  useEffect(() => {
+    const getProfile = () => Identity
+      .getUserProfile()
+      .then((profileObject) => {
+        const { email: loggedInEmail } = profileObject;
```

Comment:
multiple network calls potentially. "loggedIn" is local. put in the subs call only updating the fields you pass. 

check to ensure getUserProfile is network or local? 

have to make sure edited fields cause update at the top level if we move this getUserProfile from the top-level. if we managed the state from within the component, though, we wouldn't need to edit. 

local changes are the primary concern. because the

Reply:
updated